modules/notices/oper.py

```diff
@@ -1,7 +1,9 @@
 import re
 import fido
 
-regex = re.compile('.* Failed OPER attempt by (?P<nick>[A-Za-z0-9_´|\[\]]*) \((?P<user>[A-Za-z0-9_´|\[\]]*)@(?P<host>[A-Za-z0-9.:_\-]*)\) \[(?P<reason>[0-9.:A-F-a-f]*)\]')
+regex = re.compile('.* Failed OPER attempt by (?P<nick>[A-Za-z0-9_´|\[\]]*) '
+                   '\((?P<user>[A-Za-z0-9_´|\[\]]*)@(?P<host>[A-Za-z0-9.:_\-]*)\) '
+                   '\[(?P<reason>[0-9.:A-F-a-f]*)\]')
```

Comment:
While this is quite long and not really readable, we do not need this line break either.